packages/pie/src/hooks.ts

```diff
@@ -20,6 +20,9 @@ import {
     CommonPieProps,
 } from './types'
 
+const idIsActive = (id: DatumId, activeId: null | DatumId | DatumId[]) =>
```

Comment:
Rather than supporting a scalar or an array, I think it would be better to rename the property to `activeIds` and to support arrays exclusively.

It would also be nice to add tests about this new feature and to adjust the documentation.

Reply:
I understand, but it makes the code and typing more complex, I'd prefer to have something a bit simpler to maintain, also the name of the property is now confusing IMHO.

Reply:
do you think it's a better idea to keep `activeId` as it was, and add a new `additionalActiveIds ([])`?
so we don't break existing charts